templates/_linux.yaml

```diff
@@ -44,6 +44,10 @@ objects:
 
           cpu:
             cores: {{ item.cores }}
+{% if not osinfoname | regex_search('win') %}
```

Comment:
What is the meaning of this? The include here is only used for linux guests.

Reply:
Windows has its own template file (so does ubuntu btw, make sure it uses sockets and threads as well).

Reply:
According to this line https://github.com/kubevirt/common-templates/blob/master/templates/ubuntu.tpl.yaml#L17 Ubuntu should use generic template as well. Or am i wrong?

Reply:
No you are right, Ubuntu is fine then.